src/components/Slider/SliderBase/SliderRail.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from "react";
+import PropTypes from "prop-types";
+import { bem } from "../SliderCommons";
+
+const SliderRail = forwardRef(({ className, children, onClick }, ref) => {
```

Comment:
Is this name is standard? Maybe consider to rewrite is to SliderClickSurface?

Reply:
This name is not actual standard. But pretty wildly used in different realisations.
More then that it not only for Click it organize `Rail` for Track and FilledTrack